packages/server-wallet/src/utilities/signatures.ts

```diff
@@ -14,10 +14,18 @@ export function signState(state: State, privateKey: string): {state: State; sign
     throw new Error("The state must be signed with a participant's private key");
   }
 
-  const signature = wasmUtils.signState(toNitroState(state), privateKey).signature;
+  const signature = wasmUtils.signState(
+    toNitroState(state) as any, // see https://github.com/statechannels/native-utils/issues/28
```

Comment:
This is probably a matter of preference but I like flagging comments like these with `TODO: ` for easy searching.

Reply:
1de8960